browser/eb_application.py

```diff
@@ -265,6 +268,35 @@ def create_connection(_):
         print(err)
     return conn
 
+def initial_connection(_):
```

Comment:
Why define it with args you're not using? Just delete the `_` and remove `"caliban.db"` from the places where its called.